lib/cli/parse-args.js

```diff
@@ -59,7 +59,10 @@ module.exports = (
     if (string.has(paramName) || multiple.has(paramName)) {
       if (isBoolean) {
         if (isHelpRequest) return result;
-        throw new ServerlessError(`Unexpected boolean "--${paramName}"`, 'MISSING_CLI_PARAM_VALUE');
+        throw new ServerlessError(
```

Comment:
I'm honestly not sure about it. How are we sure that there's actually a missing value for a CLI parameter and not an unexpected boolean flag being passed?

Reply:
@pgrzesik I don't know if that's the case here, but maybe it is.

I did a quick check and I get a different error message:

<img width="731" alt="image" src="https://user-images.githubusercontent.com/720328/168987924-e2b7bdbf-168f-4579-9e77-5c586efe14b9.png">

Reply:
oh nice 👍